src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -6,7 +6,6 @@
 import java.util.Optional;
 import taxi.lib.Dao;
 import taxi.model.Car;
-import taxi.model.Driver;
 import taxi.storage.Storage;
 
 @Dao
```

Comment:
@AntonPanchishko I am confused. Why do you need two same classes `CarDaoImpl` in two different packages: `taxi.dao` and `taxi.dao.impl`?